CHANGELOG.md

```diff
@@ -2,6 +2,36 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.3.2] - 2016-05-12
+
+### Added
+- Added support for `Nexus N8k`
+  - https://github.com/cisco/cisco-network-puppet-module/tree/release_1.3.2#resource-platform-support-matrix
```

Comment:
I added this entry for n8k suport, but once we collapse to master we need to change it to point to master instead of the release branch.